app/helpers/article_helper.rb

```diff
@@ -11,4 +13,21 @@ def display_comment_timestamp(comment)
       end
     end
   end
+
+  def report_comment_link(comment)
+    mail_to REPORT_COMMENT_EMAIL_ADDRESS,
+            t('.report_comment'),
+            subject: "Reported comment in article '#{comment.article.title}'",
+            body: report_comment_email_template(comment)
+  end
+
+  private
+
+  def report_comment_email_template(comment)
```

Comment:
I dunno, for just three lines?

With the crimes against programming that already exist in this codebase, it doesn't feel like that big a deal.